src/modules/roc_core/target_posix/roc_core/time.cpp

```diff
@@ -49,6 +52,117 @@ void sleep_for_ms(uint64_t ms) {
         }
     }
 }
+#elif defined(__APPLE__) && defined(__MACH__)
+
+#include <mach/clock.h>
+#include <mach/clock_types.h>
+#include <mach/mach.h>
+#include <mach/mach_time.h>
+
+#define CLOCK_MONOTONIC SYSTEM_CLOCK
+
+uint64_t timestamp_ms() {
+    clock_serv_t cclock;
+    mach_timespec_t mts;
+
+    // Receive a permission to send a messages to a specified clock service.
+    kern_return_t ret =
+        host_get_clock_service(mach_host_self(), CLOCK_MONOTONIC, &cclock);
```

Comment:
I think so, because they are system calls, I get rid of them in: 2ae9b9ce57ef6afde2bc29c6521441dfa23b2430